app/livechat/server/index.js

```diff
@@ -77,6 +78,10 @@ import './lib/RoutingManager';
 import './lib/routing/External';
 import './lib/routing/ManualSelection';
 import './lib/routing/AutoSelection';
+import './lib/ScreenSharingManager';
+import './lib/screenSharing';
+import './lib/screenSharing/IScreenShareProvider';
```

Comment:
You don't need to import an Interface here. You just need to import it in the files that will use it.